integration-test/content-scripts/gpc.js

```diff
@@ -1,91 +0,0 @@
-/**
```

Comment:
I think the remaining content-scripts integration tests (that as far as I can tell aren't ever used) should be either deleted or moved inside of the legacy directory. So something like `integration-test/legacy/content-scripts`, just so it's clear they're old cruft.